plugin/db/mysql/mysql.go

```diff
@@ -148,13 +150,35 @@ func (driver *Driver) getVersion(ctx context.Context) (string, error) {
 
 // Execute executes a SQL statement.
 func (driver *Driver) Execute(ctx context.Context, statement string) error {
+	statements, err := bbparser.SplitMultiSQL(bbparser.MySQL, statement)
+	if err != nil {
+		return errors.Wrapf(err, "failed to split SQL statements")
+	}
+	var buf bytes.Buffer
+	delimiter := `;`
+	for _, singleSQL := range statements {
+		stmt := singleSQL.Text
+		if bbparser.IsDelimiter(stmt) {
+			delimiter, err = bbparser.ExtractDelimiter(stmt)
+			if err != nil {
+				return errors.Wrapf(err, "failed to extract delimiter")
+			}
+			continue
+		}
+		if delimiter != ";" {
+			// Trim delimiter
+			stmt = fmt.Sprintf("%s;", stmt[:len(stmt)-len(delimiter)])
+		}
+		_, _ = buf.WriteString(stmt)
```

Comment:
handle err?

Reply:
It seems that WriteString() always return nil err

Reply:
Done.